PRS/plink2_docker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# default ubuntu version: 20.04
+ARG UBUNTU_VERSION=20.04
```

Comment:
Why use a much older image for a new docker image? I think 24.04 is the latest LTS image. Unless any of the packages break in newer versions?